requirements/base.txt

```diff
@@ -7,23 +7,26 @@
 amqp==1.4.9               # via kombu
 anyjson==0.3.3            # via kombu
 billiard==3.3.0.23        # via celery
-celery==3.1.26.post2      # via django-celery
+celery==3.1.26.post2      # via django-celery, edx-celeryutils
 certifi==2019.9.11        # via requests
 chardet==3.0.4            # via requests
-django-celery==3.3.1      # via super-csv
+django-celery==3.3.1      # via edx-celeryutils, super-csv
 django-crum==0.7.3        # via super-csv
-django-model-utils==3.2.0
-django==1.11.24
+django-model-utils==3.0.0
```

Comment:
we went to an older version? 3.0.0 vs 3.2.0 ?!

Reply:
That is weird.... I did a lot of swapping virutualenvs while working on this, maybe it's a python 
2 vs 3 thing